### PR TITLE
New Package: tastyworks.tastytrade version 2.3.1

### LINKED ISSUE #111718: [Update Request]: tastyworks/tastytrade
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

tastyworks.tastyworks

### Package Version

2.3.1

### Please describe the changes you would like to see

I believe #95553 is relevant, but tastyworks has rebranded to tastytrade, and I am unsure as to how this should be handled.
The new version is: https://download.tastytrade.com/desktop-2.x.x/2.3.1/tastytrade-2.3.1.msi

### PR DESCRIPTION
**Pull request by Trenly**
* Related to #111749
* Resolves #111718 
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111750&drop=dogfoodAlpha

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
It look like you were max length with the schema. .

**Comment by Trenly** [comment]
@stephengillie - This needs a manual review using the [latest 1.5 release](https://github.com/microsoft/winget-cli/releases/tag/v1.5.1881) since the pipelines haven’t been updated with the new client yet

File: manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
```
Comment (BrandonWanHuanSheng) [inline manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.installer.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
```

File: manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
```
Comment (BrandonWanHuanSheng) [inline manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.locale.en-US.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
```

File: manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# PurchaseUrl:
+InstallationNotes: |-
+  tastytrade Desktop requires an active tastytrade Account and will not work without it.
+
+  tastytrade software connects to tastytrade services over SSL HTTPS on ports 443 and ports 7301-7305 for market data.
+  Please be sure these ports are allowed through your firewall.
+  System response and access times may vary due to trading volume, market conditions, system performance, and other factors.
+# Documentations:
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (BrandonWanHuanSheng) [inline manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.locale.en-US.yaml:35]:
```suggestion
ManifestVersion: 1.5.0
```

File: manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
```
Comment (BrandonWanHuanSheng) [inline manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
```

File: manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+PackageIdentifier: tastyworks.tastytrade
+PackageVersion: 2.3.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (BrandonWanHuanSheng) [inline manifests/t/tastyworks/tastytrade/2.3.1/tastyworks.tastytrade.yaml:8]:
```suggestion
ManifestVersion: 1.5.0
```

**Comment by BrandonWanHuanSheng** [comment]
I just testing in draft to see. If the pipelines still happening. But I actually got a validation succeed coming from 
winget validate --manifest "F:\manifests\t\tastyworks\tastytrade\2.3.1"
Manifest validation succeeded.

F:\>winget
Windows Package Manager v1.5.1881

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Manifest Error: Schema Error [Schema validation failed. Error context: <root>[InstallationNotes] Description: String should be no more than 256 characters in length. Error context: <root> Description: Failed to validate against schema associated with property name 'InstallationNotes'. ] File: tastyworks.tastytrade.locale.en-US.yaml 

It appears that the `InstallationNotes` are too long for the current version.

**Comment by Trenly** [comment]
> Automatic Validation ended with:
> 
> > Manifest Error: Schema Error [Schema validation failed. Error context: [InstallationNotes] Description: String should be no more than 256 characters in length. Error context:  Description: Failed to validate against schema associated with property name 'InstallationNotes'. ] File: tastyworks.tastytrade.locale.en-US.yaml

@stephengillie - I'm aware of that; This has been resolved in the latest version of the CLI

**Review by stephengillie — APPROVED** [review]

**Pull request merged**